paddle/fluid/pybind/pybind.cc

```diff
@@ -414,8 +414,21 @@ All parameter, weight, gradient are variables in Paddle.
              self.set_falsenet(net.Clone());
            });
 
+  py::class_<ExecutorPrepareContext>(m, "ExecutorPrepareContext");
+
   py::class_<framework::Executor>(m, "Executor")
       .def(py::init<const platform::Place &>())
+      .def_static("prepare",
```

Comment:
_prepare?

Reply:
I think the visibility can be controlled on the Python side.